pkg/releasesync/releasesync.go

```diff
@@ -0,0 +1,232 @@
+package releasesync
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/openshift/sippy/pkg/db"
+	"github.com/openshift/sippy/pkg/db/models"
+	"k8s.io/klog"
+)
+
+type releaseSyncOptions struct {
+	db            *db.DB
+	httpClient    *http.Client
+	releases      []string
+	architectures []string
+}
+
+func Import(dbc *db.DB, releases, architectures []string) error {
+	o := releaseSyncOptions{
+		db:            dbc,
+		releases:      releases,
+		architectures: architectures,
+		httpClient:    &http.Client{Timeout: 60 * time.Second},
+	}
+
+	return o.Run()
+}
+
+func (r *releaseSyncOptions) Run() error {
+	for _, release := range r.releases {
+		fmt.Fprintf(os.Stderr, "Fetching release %s from release controller...\n", release)
+		allTags := r.fetchReleaseTags(release)
+
+		for _, tags := range allTags {
+			for _, tag := range tags.Tags {
+				c := int64(0)
+				r.db.DB.Table("release_tags").Where("\"releaseTag\" = ?", tag.Name).Count(&c)
```

Comment:
Are those escaped quotes required?

Reply:
Camel case column names needs to be quoted for postgres, I should just fix this to use snake case to match the other tables.